crates/wallet/src/wallet/coin_selection.rs

```diff
@@ -1500,15 +1530,16 @@ mod test {
             vec![],
             utxos,
             FeeRate::from_sat_per_vb_unchecked(10),
-            500_000,
+            Amount::from_sat(500_000),
             &drain_script,
             &mut thread_rng(),
         );
 
+        const EXPECTED_AMOUNT: Amount = Amount::from_sat(300_000);
         assert_matches!(
             selection,
             Err(InsufficientFunds {
-                available: 300_000,
+                available: EXPECTED_AMOUNT,
```

Comment:
Can we also use a match guard here?
```suggestion
                available,
                ..
            })
            if available.to_sat() == 300_000
```

Reply:
Great suggestion, I should have known there was a better way to do this.  Fixed.